lib/riak/connection.ex

```diff
@@ -3,13 +3,21 @@ defmodule Riak.Connection do
   Create a linked process to talk with the riak server on host:port.
   """
   def start_link(host \\ '127.0.0.1', port \\ 8087, args \\ []) do
+    init_seed()
     :riakc_pb_socket.start_link(host, port, args)
   end
 
   @doc """
   Create a process to talk with the riak server on host:port.
   """
   def start(host \\ '127.0.0.1', port \\ 8087, args \\ []) do
+    init_seed()
     :riakc_pb_socket.start(host, port, args)
   end
+
+  defp init_seed do
+    # Let's initialize a seed for pseudo uniform random number - no periodic
+    << a :: 32, b :: 32, c :: 32 >> = :crypto.rand_bytes(12)
```

Comment:
No, you are right! I'll remove it.

> Le 7 avr. 2016 à 18:30, Drew Kerrigan <notifications@github.com> a écrit :
> 
> In lib/riak/connection.ex:
> 
> >      :riakc_pb_socket.start(host, port, args)
> >    end
> > +
> > +  defp init_seed do
> > +    # Let's initialize a seed for pseudo uniform random number - no periodic
> > +    << a :: 32, b :: 32, c :: 32 >> = :crypto.rand_bytes(12)
> With the timeout mechanism in place, it doesn't seem like we need to mess with the random number seed anymore right? Or were you leaving this here so that the downstream pooler stuff benefits it?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>